test/unit/links.test.js

```diff
@@ -18,8 +18,20 @@ describe('LinkParser', () => {
       fixes: ['https://github.com/nodejs/node/issues/16504'],
       refs: []
     }, {
+      // Parse non-GitHub refs.
+      // https://github.com/nodejs/node/pull/17107
       fixes: [],
       refs: ['https://en.wikipedia.org/w/index.php?title=IPv6_address&type=revision&diff=809494791&oldid=804196124']
+    }, {
+      // Parse npm update pull requests.
+      // https://github.com/nodejs/node/pull/42382
+      fixes: [],
+      refs: []
+    }, {
+      // Contains `Fixed: v8:11389` which should be ignored.
```

Comment:
Note, it's entirely possible that if we ever added an [autolink reference](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources) for `v8:<num>` then this would be converted to a link by GitHub, in which case it would be added to the metadata.

Reply:
Ignore that, we already did add autolinks to V8 references: https://github.com/nodejs/node/issues/30238#issuecomment-549162770
https://github.com/nodejs/node/pull/37276 was created after that happened.